bin/current-account-config-name.sh

```diff
@@ -4,4 +4,8 @@
 # infra/accounts that matches "<account name>.<account id>.s3.tfbackend".
 # The config name is "<account name>.<account id>""
 set -euo pipefail
-ls -1 infra/accounts | grep "$(./bin/current-account-id.sh)" | grep s3.tfbackend | sed 's/.s3.tfbackend//'
+CURRENT_ACCOUNT_ID="$(./bin/current-account-id.sh)"
```

Comment:
I don't think you need quotes outside of `$(...)`. See https://github.com/navapbc/template-infra/pull/420/files#diff-ba5902e6b742b133f9ad6ff5a8f3561c3515faff721e2e05461a6d484166d7cfR20

Reply:
oh yeah weird. in some places it tells me i need it and in others it doesn't. i'll go through and only keep the ones that are needed